src/styleguide/_patterns/02-molecules/02-locations/location-card.mustache

```diff
@@ -0,0 +1,17 @@
+<div class="grav-c-location-card">
+  <div class="grav-c-location-card__img">
+    <img src="{{location.img}}" width="425" height="300" alt="">
```

Comment:
It doesn't seem like `width` and `height` attributes are needed at all.
You might also want  to fill in the `alt` attribute with some random copy with {{location.city}} ?

Reply:
In this case `height` and `width` probably won't make any difference, added out of habit. Can add some alt text in the data